heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraScheduler.java

```diff
@@ -186,7 +186,7 @@ protected String formatJavaOpts(String javaOpts) {
     auroraProperties.put("COMPONENT_RAMMAP", Runtime.componentRamMap(runtime));
     auroraProperties.put("COMPONENT_JVM_OPTS_IN_BASE64",
         formatJavaOpts(TopologyUtils.getComponentJvmOptions(topology)));
-    auroraProperties.put("TOPOLOGY_PACKAGE_TYPE", Context.topologyPackageType(config));
+    auroraProperties.put("TOPOLOGY_PACKAGE_TYPE", Context.topologyPackageType(config).toString());
```

Comment:
We don't want to rely on `toString()` here since that should only ever really be used for logging. If we need a string better to call `.name()` on the enum, or create a `value` field in the enum that is defined as part of the enum (i.e., `TAR("tar")`).